app/models/article.rb

```diff
@@ -416,6 +416,17 @@ def access_by?(user)
     user.admin? || user_id == user.id
   end
 
+  def merge_with(other_article_id)
+    Article.find(other_article_id).comments.each do |comment|
```

Comment:
There is a lot of method chaining going on here. Is there a better way to get the comments associated without iterating through each one?